addons/binding/org.openhab.binding.innogysmarthome/.classpath

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```

Comment:
Hi @htreu, I moved the client library directly into the binding. There are several reasons for having it separated, but you are absolutely right, that it makes the binding development more difficult.
I also switched to the internal libraries, but regarding google http client, I use this because of the automatic handling of oauth credential refreshing and like to stick to it. I hope this is okay for now.